src/components/lightOrDarkMode.jsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import IconButton from "@mui/material/IconButton";
+import Box from "@mui/material/Box";
+import { useTheme, ThemeProvider, createTheme } from "@mui/material/styles";
+import Brightness4Icon from "@mui/icons-material/Brightness4";
+import Brightness7Icon from "@mui/icons-material/Brightness7";
+import { CssBaseline } from "@mui/material";
+
+const ColorModeContext = React.createContext({ toggleColorMode: () => {} });
+
+function LightOrDarkMode() {
+  const theme = useTheme();
+  const colorMode = React.useContext(ColorModeContext);
+  return (
+    <Box
```

Comment:
I think the Box is going to take over the DOM. I tried to use this within the Authenticator component and I expect to see a signout button and this toggle. However, what I ended up seeing is that the signout button disappears and the toggle button is the only thing rendered.

The functionality works. However, lets make sure that we can use it within the Authenticator component as that is wat is going to be the Higher order Component

Reply:
I removed the Box from the component and the dark mode toggle is on the navbar for now.